tests/ui/structs/default-field-values/non-exhaustive-ctor.rs

```diff
@@ -0,0 +1,26 @@
+#![feature(default_field_values)]
```

Comment:
Suggestion: since the suggestions are machine-applicable, could you add `//@ run-rustfix` or `//@ rustfix-only-machine-applicable`?

Reply:
Ah ok, thats fine